### PR TITLE
Creates SIG "federated" and the SIG charter.

### PR DESCRIPTION
**Pull request by krzys-ostrowski**
Creates a new SIG named "federated" for collaborations around TensorFlow Federated (TFF), and adds the initial SIG charter (CHARTER.md).

### CODE REVIEW

**Comment by ematejska** [comment]
Waiting for approval until RFC for SIG Federated is merged.

**Review by theadactyl — APPROVED** [review]

**Pull request merged**
